scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Packaging scripts
+
+This directory contains useful packaging scripts.
+
+## `configure-hypervisor.sh`
+
+This script generates the official set of hypervisor build configuration
```

Comment:
When you say, "in the text", do you mean the script header or the README? The former mentions qemu and the latter gives and example. I'm open to suggestions here ;)